docs/design/UITesting.md

```diff
@@ -10,60 +14,36 @@ Appium relies on different implementations called `drivers` for each platform th
 * iOS       - [XCUITest](https://github.com/appium/appium-xcuitest-driver)
 * Android   - [UIAutomator2](https://github.com/appium/appium-uiautomator2-driver)
 
-## Creating a new test
-
-### The sample project
-
-The project that is used for UI Tests is located here: `src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj`
 
-There are two types of tests you can add, Issue and Gallery.
+## Creating a new test
 
 ### Adding a new Issue
 
 This will be the majority of new tests added which will be primarily for testing functionality and adding regression tests.
 
-You will need to create some kind of UI to test against, which will go in the Controls.Sample.UITests project. Create a new class and attribute it with `[Issue]` and derive from `TestContentPage`.
-
-Then in the Controls.AppiumTests project add a new class that derives from `_IssuesUITest` and add your test.
-
-You can use the example for the sample project [here](https://github.com/dotnet/maui/blob/main/src/Controls/tests/TestCases.HostApp/Issues/RefreshViewPage.cs) and the example for the corresponding test [here](https://github.com/dotnet/maui/tree/main/src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/RefreshViewTests.cs).
-
-### Adding a GalleryPage
+You will need to create some kind of UI to test against, which will go in the Controls.TestCases.HostApp project. Create a new class within `src/Controls/tests/TestCases.HostApp/Issues` and attribute it with `[Issue]` and derive from `TestContentPage` (or `TestNavigationPage`, or `TestShellPage`, and so on).
```

Comment:
The `Test*` Pages are kind of just hold overs from old code

I would just say that they can create a normal page like they would inside any reproduction app. 

These can be in Xaml or just code and then include a screen shot.

Can you add a note that this is the actual MAUI app under test? You could even mention how this is just a MAUI app and that it doesn't have anything special required that relates to Appium